### DISCUSSION TITLE
using ctrl-shift-enter to source R script with echo=TRUE

### DISCUSSION THREAD

**Discussion by benzipperer** [posit-dev/positron#5185] [Q&A] [answered]
When editing `my_script.R`, the Ctrl-Shift-Enter keybinding runs `source("my_script.R")` . 

Is it instead possible to choose the default RStudio behavior of `source("my_script.R", echo=TRUE)` in order to see the input and output?

**Comment by lionel-** [comment, marked as answer]
Thanks for the suggestion! We should match RStudio's behaviour here. I opened an issue to track this: https://github.com/posit-dev/positron/issues/5191
